package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@sern/handler",
     "packageManager": "pnpm@7.28.0",
     "version": "2.6.0",
-    "description": "A customizable, batteries-included, powerful discord.js framework to automate and streamline bot development.",
+    "description": "A complete, customizable, typesafe, & reactive framework for discord bots.",
```

Comment:
I don't think we should say "for discord bots"

Reply:
To keep it simple, and short.

Reply:
Any alternatives?

Reply:
It's fine, just remove "for discord bots."

Reply:
its more up to you, and how you like it